web/src/components/predictions-components/interval-choice.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { Box } from "theme-ui";
+import { Range, getTrackBackground } from 'react-range';
+
+const STEP = 0.1;
+
+function IntervalChoice(props) {
+  const lower = props.lower;
+  const upper = props.upper;
+  const [values, setValues] = useState([lower, upper]);
+
+  const afterSubmission = (event) => {
+    event.preventDefault();
+  };
+
+  return (
+    <form onSubmit={event => afterSubmission(event)}>
+      <Box mt={1} mx={3}>
+        <Range
+          draggableTrack
+          values={values}
+          step={STEP}
+          min={lower}
+          max={upper}
+          onChange={values => setValues(values)}
+          renderTrack={({ props, children }) => (
```

Comment:
I have a slight preference for defining Track and Thumb as separate components within this file... let me know what you think!

Reply:
Thumb was implementable as a component but was having some trouble with Track and the ref prop

Reply:
actually, just figured out a fix